great_expectations/checkpoint/util.py

```diff
@@ -410,9 +410,6 @@ def validate_validation_dict(validation_dict: dict, batch_request_required: bool
     if not validation_dict.get("expectation_suite_name"):
         raise gx_exceptions.CheckpointError("validation expectation_suite_name must be specified")
 
-    if not validation_dict.get("action_list"):
-        raise gx_exceptions.CheckpointError("validation action_list cannot be empty")
-
```

Comment:
Removing this frees up a lot of errors within the test suite.

Question for V1 - does this hold? Is a checkpoint without an action valid? I say yes you don't need to configure one.

Reply:
Yes, we don't need to configure actions. Checkpoints would still have value in group validations and running them. These results would be persisted without an action.

Reply:
I believe you still need to pass an empty list in the V1 checkpoint. Should we allow users to omit that? I'd think yes (i.e. `Field(default_factory=list)`)

Reply:
With pydantic you can also use `[]`. It will make a deep copy.